### PR TITLE
Add latest component behavior standards

### PR DESCRIPTION
**Pull request by jhorbulyk**
There was an internal discussion at e.io where we went through these ideas and produced a new/revised approach for presenting them.

### CODE REVIEW

**Comment by philecs** [comment]
I'll review at latest tomorrow

File: Adapters/AdapterBehaviorStandardization/StandardizedActionsAndTriggers.md [hunk 1]
```diff
@@ -1,547 +1,371 @@
...
-**Semantic Version of Document: 1.0.0**
+**Semantic Version of Document: 2.0.0**
 
 # Table of Contents
-- [Standardized Triggers (including webhooks)](#standardized-triggers-including-webhooks)
-  * [Bulk Extract](#bulk-extract)
-  * [Get Objects Polling](#get-objects-polling)
-  * [Get Objects Webhook](#get-objects-webhook)
-  * [Get Deleted Objects Polling](#get-deleted-objects-polling)
-  * [Get Deleted Objects Webhook](#get-deleted-objects-webhook)
```
Comment (philecs) [inline Adapters/AdapterBehaviorStandardization/StandardizedActionsAndTriggers.md]: Why remove the webhook descriptions?

**Comment by jhorbulyk** [comment]
> I liked the general split between triggers and actions that we had before. This helped to get an overview of all standardized triggers (same for actions).

Sure I could re-add that.  However, this time, I will put actions before triggers.

File: Adapters/AdapterBehaviorStandardization/StandardizedActionsAndTriggers.md (hunk 1 above)
Comment (jhorbulyk) [reply Adapters/AdapterBehaviorStandardization/StandardizedActionsAndTriggers.md]: It's still there, just in the non-standardized section.  The reason for that is that webhook is that the general webhook case is receive data and pass it along.

Comment (philecs) [reply Adapters/AdapterBehaviorStandardization/StandardizedActionsAndTriggers.md]: I saw that it is listed there but we had a description for what it is and what the differences with regards to polling triggers are. Couldn't find this information in the new version.

Comment (jhorbulyk) [reply Adapters/AdapterBehaviorStandardization/StandardizedActionsAndTriggers.md]: I added that information here: https://github.com/openintegrationhub/Connectors/pull/40

**Comment by philecs** [comment]
Looks good in general, but the TOC has some information in it (such as webhook trigger) that is not present in the document. We should at least write 1 or 2 sentences for each entry in the TOC (or remove it from TOC if not needed there)

**Comment by jhorbulyk** [comment]
I can go and add a simple sentence for every missing entry.

**Comment by jhorbulyk** [comment]
> I can go and add a simple sentence for every missing entry.

Done

**Pull request merged**